packages/flutter/test/cupertino/activity_indicator_test.dart

```diff
@@ -77,60 +77,6 @@ void main() {
     );
   });
 
-  testWidgets('Activity indicator with iOS14 style',
```

Comment:
I think it's still valuable to keep this test around.  Just leave off `iOSVersionStyle` so it defaults to iOS14?

Reply:
The original tests are now default to iOS14 so we nolonger need these iOS14 tests.
But should we add some tests for iOS13 style?

Reply:
Oh I see.  No I think it's fine since iOS13 will be removed in the future.

Alright this looks good and should be merged when our build is green.  Thanks for following up with this work!